frontend/src/index.css

```diff
@@ -1,6 +1,7 @@
 body {
 	margin: 0;
 	min-height: 100vh;
+    overflow: hidden;
```

Comment:
It seems that it needs running code formatting.

Reply:
I made a mistake. I finished formatting the code.